src/main/java/ru/sbtqa/tag/apifactory/ApiEntry.java

```diff
@@ -448,6 +462,25 @@ private void setDependentResponseParameters() throws ApiException {
         }
     }
 
+    private Map<String, Object> sortByKeyLength(Map<String, Object> map) {
+        Map<String, Object> treeMap = new TreeMap<>(
+                new Comparator<String>() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1604)